cmapi/cmapi_server/handlers/cluster.py

```diff
@@ -141,7 +141,8 @@ def start(
     @staticmethod
     def shutdown(
         config: str = DEFAULT_MCS_CONF_PATH,
-        logger: logging.Logger = logging.getLogger('cmapi_server')
+        logger: logging.Logger = logging.getLogger('cmapi_server'),
+        timeout: int = 10
```

Comment:
Units used are unclear to me.

Reply:
10 seconds looks too short as a timeout for shutdown.

Reply:
Changed to 15 seconds. Added units to docstring. Anyway it's only for next releases.